tests/runner.py

```diff
@@ -1399,7 +1399,7 @@ def run_browser(self, html_file, message, expectedResult=None, timeout=None):
       print('(moving on..)')
 
   def with_report_result(self, code):
-    return '#define EMTEST_PORT_NUMBER %d\n#include "%s"\n' % (self.port, path_from_root('tests', 'report_result.h')) + code
+    return '#define EMTEST_PORT_NUMBER %d\n#include "%s"\n#include "%s"\n' % (self.port, path_from_root('tests', 'report_result.h'), path_from_root('tests', 'report_result.cpp')) + code
```

Comment:
Including the .cpp file like this also seems strange.

Reply:
I agree it does look somewhat strange. I refactored this now to clarify the various pieces, and to embed the cpp there, let me know if that's clearer.